<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>294</width>
    <height>439</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Формат даты</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Введите дату:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="le_date">
      <property name="inputMask">
       <string>00.00.0000</string>
      </property>
      <property name="placeholderText">
       <string>00.00.0000</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>День:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="le_day"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Месяц:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="le_month"/>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Год:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="le_year"/>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Формат дд.мм.гггг</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbl_format_type1">
      <property name="text">
       <string>QLabel lbl_format_type1</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Формат мм/дд/гггг</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbl_format_type2">
      <property name="text">
       <string>QLabel lbl_format_type2</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Формат: дд месяц гггг</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbl_format_type3">
      <property name="text">
       <string>QLabel lbl_format_type3</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lbl_message">
      <property name="text">
       <string>QLabel lbl_message</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
